packages/language-server/test/completion-items-spec.ts

```diff
@@ -40,6 +40,15 @@ describe("the UI5 language assistant Code Completion Services", () => {
     return result?.[1];
   }
 
+  /** Return the attributes from a tag name suggestion insert text  */
+  function getAttributes(textEdit: TextEdit | undefined): string[] {
```

Comment:
if you ever need more accurate parsing you can
use the XML Parser from XML-Tools to parse only an attribute's text.
It is not limited to starting at the root